prepare-tools-action/action.yml

```
@@ -4,12 +4,12 @@ inputs:
   operator-sdk-version:
     description: Version of operator-sdk binary
     required: false
-    default: v1.21.0
+    default: v1.23.0
   opm-version:
     description: Version of opm binary
     required: false
     # see https://github.com/operator-framework/operator-sdk/blob/v1.21.0/go.mod#L21
```

```suggestion
    # see https://github.com/operator-framework/operator-sdk/blob/v1.23.0/go.mod#L21
```
and I believe we should rename the `opm-version` var to something like `operator-registry` to avoid confusion with the link target and rest of the script 
```
curl -Lo opm https://github.com/operator-framework/operator-registry/releases/download/${{ inputs.opm-version }}/linux-amd64-opm
```

---

@xcoulon done in [65778e1](https://github.com/codeready-toolchain/toolchain-cicd/pull/76/commits/65778e1cdd595bf651384383edffdedcd577526d)

---

thanks!